src/components/homepage/Carousel.js

```diff
@@ -26,15 +26,18 @@ const CarouselWide = () => {
   }, []);
 
   const mapCars = indexes.map(
-    (index) => (
-      <CarCard
-        key={cars[index].id}
-        id={cars[index].id}
-        name={cars[index].name}
-        model={cars[index].model}
-        brand={cars[index].brand}
-        imageUrl={cars[index].image_url}
-      />
+    (index) => (cars[index].removed
+      ? (
+        <CarCard
+          key={cars[index].id}
+          id={cars[index].id}
+          name={cars[index].name}
+          model={cars[index].model}
+          brand={cars[index].brand}
+          imageUrl={cars[index].image_url}
+        />
+      )
+      : null
```

Comment:
- When I login I should be presented with the cars list, unfortunately I am not. This seems to be the cause.